charts/nr-ebpf-agent/values.yaml

```diff
@@ -1,58 +1,95 @@
+---
 # -- Name of the Kubernetes cluster to be monitored. Mandatory. Can be configured with `global.cluster`
 cluster: ""
 # -- The license key to use. Can be configured with `global.licenseKey`
 licenseKey: ""
-# -- Endpoint to export data to via the otel collector. NR prod (otlp.nr-data.net:4317) by default. Staging (staging-otlp.nr-data.net:4317) otherwise. Requires a valid staging license key. Can also be configured with global.nrStaging.
+# -- In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there. Can be configured also with `global.customSecretName`
+customSecretName: ""
+# -- In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey`
+customSecretLicenseKey: ""
+# Endpoint to export data to via the otel collector. NR prod (otlp.nr-data.net:4317) by default.
+# Staging (staging-otlp.nr-data.net:4317) otherwise.
 nrStaging: false
-# -- Configures the agent to send all data through the proxy specified via the otel collector.
+# Configures the agent to send all data through the proxy specified via the otel collector.
 proxy: ""
-# -- Drop data when service names map to an IP address.
+# Drop data where to service names mapps to an IP address
 dropDataIpServiceNames: true
-# -- Drop data from the kube-system namespace.
+# Drop data from to the kube-system namespace
```

Comment:
We will need to prefix all of the relevant comments with `--`. This is because the ReadMe's auto generation relies on that prefix in order to add the description. 

The auto generation of the ReadMe is done by running [helm-docs](https://github.com/norwoodj/helm-docs).